x-pack/plugins/task_manager/server/metrics/task_metrics_collector.ts

```diff
@@ -70,6 +88,9 @@ export class TaskManagerMetricsCollector implements ITaskEventEmitter<TaskLifecy
 
   private async runCollectionCycle() {
     const start = Date.now();
+    const searchedTypes = Array.from(this.taskTypes)
+      .concat(Array.from(this.removedTypes))
```

Comment:
hmmm ... I saw this in the emergency fix PR and didn't think too much about it, but ... now I'm wondering why we are doing this.  Seems like we should exclude these, but actually I don't think we even need to since I believe we don't allow a type to be in the `REMOVED_TYPES` and registered.  Seems like we could just delete this line ...

If we're in a pinch to get this merged (needs to be done by Friday I guess, anyway), we can defer changing it till later ... again, it's not critical, seems like some code we just don't need though.

Reply:
oh okay, that makes sense. I will go ahead and remove this line

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/185892/commits/69269edb5567b763a226d1e723d876dd47f6b018